crates/nargo_cli/src/cli/init_cmd.rs

```diff
@@ -6,13 +6,15 @@ use acvm::Backend;
 use clap::Args;
 use nargo::constants::{PKG_FILE, SRC_DIR};
 use nargo::package::PackageType;
+use noirc_frontend::graph::CrateName;
 use std::path::PathBuf;
+use std::str::FromStr;
 
 /// Create a Noir project in the current directory.
 #[derive(Debug, Clone, Args)]
 pub(crate) struct InitCommand {
     /// Name of the package [default: current directory name]
-    #[clap(long)]
+    #[clap(long, value_parser = CrateName::from_str)]
```

Comment:
```suggestion
    #[clap(long)]
```
Clap does this by default if you change `Option<String>` to `Option<CrateName>`